src/shared/capture/capturev4l.cpp

```diff
@@ -581,7 +577,8 @@ CaptureV4L::CaptureV4L(VarList * _settings,int default_camera_id) : CaptureInter
     }
     
     if (cam_id > cam_list[cam_count-1]) {
-        fprintf(stderr,"CaptureV4L Error: no camera found with index %d. Max index is: %d\n",cam_id,cam_list[cam_count-1]);
+        //fprintf(stderr,"CaptureV4L Error: no camera found with index %d. Max index is: %d\n",cam_id,cam_list[cam_count-1]);   /// 2/5/16 --- shh, please be quiet
```

Comment:
In reality, this doesn't happen very often (only at startup when it attempts to enumerate cameras) but I didn't want to confuse users that start seeing responses about V4L cameras when they're used to the dc1394 or others.  A one-time message is acceptable for this -- but it will actually get printed 4 times -- once for each camera.